macos/Sources/Ghostty/SurfaceView_AppKit.swift

```diff
@@ -83,7 +83,15 @@ extension Ghostty {
         @Published private(set) var backgroundColor: Color? = nil
 
         /// True when the bell is active. This is set inactive on focus or event.
-        @Published private(set) var bell: Bool = false
+        @Published private(set) var bell: Bool = false {
+            didSet {
```

Comment:
Instead of this and tightly coupling it to the app, I think it'd make sense instead to use a notification that anyone can listen to. 

Probably AppDelegate on its own would have to listen to it.